### DISCUSSION TITLE
APT repo update missing 2.0.13

### DISCUSSION THREAD

**Discussion by jonwilliams84** [hyperion-project/hyperion.ng#1469] [Releases] [answered]
Hi, 

Trying to install the latest version 2.0.13 but APT only lists 2.0.12 being available.

The CI pipeline Build #590 appears to have run but still cannot install 2.0.13

Any ideas?

**Comment by Lord-Grey** [comment]
@jonwilliams84 Thank you for reporting. We will have a look!

**Reply by jonwilliams84** [reply]
Thanks, just verified apt now has the latest version.

**Comment by Lord-Grey** [comment, marked as answer]
@jonwilliams84 APT Server version should be correct now.
Would you mind testing, if it works?

**Reply by Germaenace** [reply]
Had (obviously) the same issue.
Now the update to 2.0.13 works like expected with `sudo apt-get update && sudo apt-get upgrade`.

FYI:
That's the output:

```
Preparing to unpack .../hyperion_2.0.13~buster_armhf.deb ...
---Hyperion ambient light prerm ---
---Hyperion ambient light preinst ---
--> stop init deamon: systemd
Unpacking hyperion (2.0.13~buster) over (2.0.12~buster) ...
Setting up hyperion (2.0.13~buster) ...
--- Hyperion ambient light postinstall ---

-----------------------------------------------------------------------------
---> Hyperion has been upgraded!
--->
---> For configuration, visit with your browser: 192.***.***.***:8090
---> or if already used by another service try: 192.***.***.***:8091
-----------------------------------------------------------------------------
Webpage: www.hyperion-project.org
Forum: www.hyperion-project.org
Documenation: docs.hyperion-project.org
-----------------------------------------------------------------------------
```

**Reply by jonwilliams84** [reply]
Yep, all good! 

I maintain sirfragalot/hyperion.ng docker image and this has now built successfully for amd64, armv7 and arm64.

Tested on, amd64, armv7 and arm64 (pi4 and Jetson nano)

All working as it should.

Nice one
